Diabetes/projects/diabetes_project_1.html

```diff
@@ -0,0 +1,87 @@
+<html>
+
+<head>
+  <script src='https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js'></script>
+  <script type='text/javascript'>
+    $(document).ready(function () {
+      $('body').append(`<p id="prediction">Prediction: <p>`);
+
+      $.fn.serializeObject = function () {
+        var object = {};
+        $.each(this.serializeArray(), function () {
+          if (object[this.name] !== undefined) {
+            [object[this.name]].push(this.value || '');
+          } else {
+            object[this.name] = this.value || '';
+          }
+        });
+        return object;
+      };
+
+      var DATA;
+      function predict(data = DATA) {
+        console.log('data', DATA);
+        var ENDPOINT = $('[name=url]').val();
+        return fetch(ENDPOINT, {
+          method: 'POST',
+          body: data,
+        })
+          .then((res) => res.json())
+          .then((resJSON) => {
+            console.log('resJSON', resJSON);
+            resJSON = JSON.parse(resJSON['body']);
+            var innerHTML = $('#prediction').filter('p').text();
```

Comment:
this is the wrong approach. what is happening here is that the code is responsible for both fetching the data and writing to the dom. Code should generally be handling a single concern. Factor out the code to update the dom into a single function that accepts arguments that you need from the api